styles/plugins/_p-ember-power-select.scss

```diff
@@ -163,12 +163,14 @@ $ember-power-select-multiple-option-line-height: 1.45;
 
 .ember-power-select-trigger:focus,
 .ember-power-select-trigger--active {
-  outline: none;
-  border-width: 0.1rem !important; // Temporary webuniversum override
-  border-color: var(
-    --au-outline-color
-  ) !important; // Temporary webuniversum override
-  box-shadow: inset 0 0 0 0.2rem var(--au-outline-color) !important; // Temporary webuniversum override
+  &:not(.ember-power-select-trigger[aria-disabled="true"]) {
+    outline: none;
+    border-width: 0.1rem !important; // Temporary webuniversum override
```

Comment:
I guess we can remove some of these overrides? I don't think there are projects that mix Appuniversum + Webuniversum.